<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineSourcesWidget</class>
 <widget class="QWidget" name="OnlineSourcesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="parserLayout"/>
     </item>
     <item>
      <widget class="QPushButton" name="checkButton">
       <property name="text">
        <string>check</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Track Artist</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="trackArtistLayout">
     <item>
      <widget class="QLineEdit" name="trackArtistEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyTrackArtistButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Album Artist</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="albumArtistLayout">
     <item>
      <widget class="QLineEdit" name="albumArtistEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyAlbumArtistButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="yearLayout">
     <item>
      <widget class="QLineEdit" name="yearEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyYearButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Album</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="albumLayout">
     <item>
      <widget class="QListWidget" name="albumList">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyAlbumButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="genreLayout">
     <item>
      <widget class="QListWidget" name="genreList">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyGenreButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Cover</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <layout class="QGridLayout" name="coverLayout">
     <item row="0" column="3">
      <widget class="QPushButton" name="applyCoverButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="coverLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLabel" name="coverInfoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="sourceCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyAllButton">
       <property name="text">
        <string>apply all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Track</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="trackList">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyTrackButton">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
